src/main/java/io/cryostat/core/net/discovery/JvmDiscoveryClient.java

```diff
@@ -26,11 +26,12 @@
 import org.openjdk.jmc.jdp.client.DiscoveryListener;
 import org.openjdk.jmc.jdp.client.JDPClient;
 
-import io.cryostat.core.log.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class JvmDiscoveryClient {
 
-    private final Logger logger;
+    private Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
Could these be kept `final`?

Reply:
hmm I believe when I kept them as finals an error was produced but I can double check that now